Gems/Atom/RHI/Vulkan/Code/Source/RHI/Vulkan.h

```diff
@@ -23,7 +23,10 @@ namespace AZ
     namespace RHI
     {
         class ScopeAttachment;
-        enum class BufferBindFlags : uint32_t;
+        inline namespace BufferBindFlagsNamespace
+        {
+            enum class BufferBindFlags : uint32_t;
+        }
```

Comment:
I had to change this forward declaration because the `AZ_ENUM_CLASS_WITH_UNDERLYING_TYPE` macro wraps the actual enum in an outer namespace (instead of AZ::RHI::BufferBindFlags we get AZ::RHI::BufferBindFlagsNamespace::BufferBindFlags).

Reply:
Can you add a comment here with this description please. Thanks